infra/resources/_modules/cdn/storage_account.tf

```diff
@@ -30,3 +30,27 @@ resource "azurerm_storage_container" "well_known" {
   # tfsec:ignore:azure-storage-no-public-access
   container_access_type = "container"
 }
+
+resource "azurerm_storage_container" "exchange" {
```

Comment:
are these blobs reachable from the public internet?

Reply:
yes

Reply:
Are these supposed to be served through a CDN? In case we should enforce access exclusively through the CDN as direct storage calls from a public web can incur additional billing.